pkg/oc/bootstrap/clusteradd/cmd.go

```diff
@@ -94,33 +108,23 @@ func (c *ClusterAddConfig) Run() error {
 			}
 
 			component := &service_catalog.ServiceCatalogComponentOptions{
-				OCImage:              c.openshiftImage(),
-				MasterConfigDir:      path.Join(c.GetKubeAPIServerConfigDir()),
-				ImageFormat:          c.imageFormat(),
 				PublicMasterHostName: masterURL.Hostname(),
```

Comment:
remove this. Make the service catalog write code to determine their info from the provided